src/ProcessMaker/Nayra/Contracts/Bpmn/ErrorInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace ProcessMaker\Nayra\Contracts\Bpmn;
+
+/**
+ * Payload for the ErrorEventDefinition
```

Comment:
Correct the interface documentation to specify that it is the ErrorInterface and not the payload of error event definition

Reply:
fixed